hugegraph-core/src/main/java/org/apache/hugegraph/masterelection/GlobalMasterInfo.java

```diff
@@ -19,26 +19,22 @@
 
 public class GlobalMasterInfo {
 
-    private boolean isMaster;
-    private String url;
-
+    private NodeInfo nodeInfo;
     private volatile boolean featureSupport;
 
     public GlobalMasterInfo() {
         this.featureSupport = false;
+        this.nodeInfo = new NodeInfo(false, "");
     }
 
-    public synchronized void set(boolean isMaster, String url) {
-        this.isMaster = isMaster;
-        this.url = url;
-    }
-
-    public synchronized boolean isMaster() {
-        return this.isMaster;
+    public final void nodeInfo(boolean isMaster, String url) {
```

Comment:
nice but why do we add final for the method？（for design？）

Reply:
> nice but why do we add final for the method？（for design？）

final method no usefull，only express dont modify

Reply:
looks useful, the subclasses cannot override it